backend/uclapi/workspaces/occupeye/utils.py

```diff
@@ -31,7 +31,12 @@ def authenticated_request(url, bearer):
 def survey_ids_to_surveys(surveys_data, survey_ids):
         if survey_ids:
             try:
-                survey_ids_list = [int(x) for x in survey_ids.split(',')]
+                # If we cast to a set then a list we remove duplicates
+                # This avoids the user specifying a filter with a bad
+                # ID and a duplicate good one, which would cause issues.
+                survey_ids_list = list(
+                    set([int(x) for x in survey_ids.split(',')])
+                )

```

Comment:
Is there any particular reason to convert back to a list? We're storing hashable elements (ints) and not using any list specific functions from what I can tell so shouldn't hit into runtime errors.

Reply:
Good question. The reason was because it was a list to start with. If we can use sets in the same way as lists then I'm happy to remove that.